cap-2.x/eks/variables.tf.json

```diff
@@ -0,0 +1,82 @@
+            "default": 3
+        },
+        "instance_type": {
+            "type": "string",
+            "description": "The type of instance used for the provisioned workers.",
+            "default": "t3.xlarge"
+        },
+        "k8s_version": {
+            "type": "string",
+            "description": "k8s version",
+            "default": "1.17"
+        },
+        "region": {
+            "type": "string",
+            "description": "The AWS region where the cluster and related resources will be created."
+        },
+        "access_key_id": {
+            "type": "string",
+            "description": "AWS access key ID. See https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys for details."
+        },
+        "secret_access_key": {
+            "type": "string",
+            "description": "AWS secret access key (password). See https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys for details."
+        },
+        "keypair_name": {
+            "type": "string",
+            "description": "Name of the EC2 Key Pair used for accessing worker nodes via SSH. See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html for details."
+
+        },
+        "admin_password": {
+            "type": "string",
+            "description":"Intial password for Cloud Foundry 'admin' user, UAA 'admin' OAuth client, and metrics 'admin' login. We recommend changing this password after deployment. See https://documentation.suse.com/suse-cap/single-html/cap-guides/#cha-cap-manage-passwords"
+        },
+        "disk_size_gb": {
+            "type": "number",
+            "description": "The worker node storage capacity. (Minimum:80, Maximum: 16383)",
+            "default": 80
+        },
+        "cluster_labels": {
+            "type": "map",
+            "description": "Tags to be applied to resources in your cluster. (Optional)"
+        },
+        "hosted_zone_name": {
+            "type": "string",
+            "description": "Name of the Route 53 Hosted zone created for the cluster."
+        },
+        "cap_domain": {
+            "type": "string",
+            "description": "The FQDN of your cluster - a subdomain of the Hosted zone."
+        },
+        "email": {
+            "type": "string",
+            "description": "Email address to send TLS certificate notifications to."
+        },
+        "cluster_tag_key": {
+            "type": "string",
+            "description": "User-provided cluster tag key",
+            "default": ""
+        },
+        "cluster_tag_value": {
+            "type": "string",
+            "description": "User-provided cluster tag value",
+            "default": ""
+        },
+        "eirini_enabled": {
+            "type": "string",
+            "description": "Eirini enabled?",
+            "default": "true"
+        },
+        "ha_enabled": {
+            "type": "string",
```

Comment:
Why not use `"type": "bool",` ?

Reply:
Terraform bools are quite awkward to use so safer to stick with strings - https://www.terraform.io/docs/configuration-0-11/variables.html#booleans